02-express-tutorial/public/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <h1> Week 3 Express </h1>
+    <button id='getAllProductsBtn'>See All Products</button>
+    <div id="productsContainer"></div>
+
+    <script>
```

Comment:
Nitpick (doesn't require change):
This script tag should have a [type="module"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type), which is a deferred JavaScript script.